src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -2154,6 +2154,14 @@ paths:
             type: string
             format: text
             minLength: 1
+        - in: query
+          name: ignore_warnings
+          description: if true, any ignorable warnings are ignored. if false (or missing),
```

Comment:
Hmmm. I feel we might need to better explain users what the warnings related to this operation could be? Like in this case, give specifics so that user understands the warnings are about registered auth certificates?

You could also instead (or, also) explain these things in the markdown documentation, but maybe the openapi definition is a pretty good place for this?

Client add says this in definition-part:

```
      description: >
        Adds new client to the system.
        Note that with this endpoint it is possible to add an unregistered member as a client.
        Attempt to add an unregistered member with ClientAdd.ignore_warnings = false causes the operation to fail with a warning in response's ErrorInfo object.
        Attempt to add an unregistered member with ClientAdd.ignore_warnings = true succeeds.
```

But unfortunately this is not consistent, other endpoints with an `ignore_warnings` possibility do not have the same type of explanation.

This ticket could add some explanation of warnings for this endpoint, or maybe documentation for all endpoints could be updated and made consistent in a separate ticket....

Reply:
I updated the OpenAPI definition for this endpoint. I think it's better to have all the related documentation in one place to make consuming the API easier for the users. In my opinion OpenAPI definition is the right place for this kind of documentation. I also think that it's a good idea to create a separate ticket about updating the OpenAPI definition and making it more consistent. More example messages should be added to the documentation too since some endpoints  have parameters which values are difficult to get right, e.g. endpoints related to CSR creation.

Reply:
Created https://jira.niis.org/browse/XRDDEV-1272